src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts

```diff
@@ -89,11 +92,18 @@ export const styles = {
       backgroundColor: 'primary.50'
     }
   },
-  componentStyles: {
-    textAlign: 'left',
-    margin: 0,
-    mb: 0
-  },
-  descriptionStyles: { typography: 'caption' },
-  titleStyles: { typography: 'h6', mb: 0 }
+  titleWithDescription: {
+    wrapper: {
+      textAlign: 'left',
+      margin: 0,
```

Comment:
shound we provide margin 0 and mb 0 after you updated titleWithDescriprion?

Reply:
fixed